src/odemis/cli/convert.py

```diff
@@ -233,6 +234,14 @@ def leader_quality(da):
             # A lot less likely
             return numpy.prod(da.shape) / 100
 
+    def get_acq_time(fn):
+        converter = dataio.find_fittest_converter(fn)
+        # TODO: use open_data/DataArrayShadow when converter support it
```

Comment:
Instead of this TODO (which is a duplicate from the code below), the right way would be to open all the files, just once, and then, order them by date.
I think you can do it right now with the following:
```
def get_acq_time(das):
    return das[0].metadata.get(model.MD_ACQ_DATE, 0)

da_streams = sorted(da_streams, key=get_acq_time)
```